src/Nexus.js

```diff
@@ -185,6 +185,6 @@ const Nexus = {
   },
 };
 
-Nexus.Mixin = Mixin(Nexus);
+Nexus.Mixin = new Mixin(Nexus);
```

Comment:
The semantics are wrong, Mixin should be renamed createMixin and called without `new`.